### DISCUSSION TITLE
Wiki typos - Image Sets

### DISCUSSION THREAD

**Discussion by psergiu** [erichelgeson/BlueSCSI-v1#168] [General]
On the Image Sets page,
https://github.com/erichelgeson/BlueSCSI/wiki/Image-Sets
 the last paragraph mentions a pin named "BA1" - was this supposed to be PB1 ?

Also, are pull-up resistors needed for the two pins ? ( due to the mention of a 3x2 block header instead of 2x2) 

Thank you !

**Comment by erichelgeson** [comment]
Yes - feel free to edit the wiki if you find typos.

It uses the internal pull up, so no resistors are needed. You could use a 2x2 but you may loose the jumper when it's in the off position (if using jumpers)
